starter-code/src/functions-and-arrays.js

```diff
@@ -1,17 +1,63 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (num1, num2){
+  console.log(Math.max(num1, num2))
+}
+
+maxOfTwoNumbers(4, 7)
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(myWords){
+  let longestWord = 0
+
+  for (let i = 0; i < myWords.length; i++){
+    if (myWords[i].length > longestWord){
+      longestWord = myWords[i]
+    }
+  return longestWord
+  //console.log(i)
+  //console.log(longestWord)
+  }
+}
+
+findLongestWord(words)
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+let result3 = 0
+
+function sumArray(myNumber){
+  for (let i=0; i < myNumber.length; i++){
+    result3 += myNumber[i]
+  }
+  return result3
+}
+sumArray(numbers)
+//console.log(result)
+
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+
+const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+
+let result4a = 0
+
+function averageNumbers(myNumbersAvg){
+  for (let i=0; i < myNumbersAvg.length; i++){
+    result4a += myNumbersAvg[i] / myNumbersAvg.length
+  }
+  return result4a
+}
+averageNumbers(numbersAvg)
+
```

Comment:
nice job, also i could say as you see we have sumArray function and we could use it to get the average but its really good nice